src/app/policies/components/PolicyList.vue

```diff
@@ -46,7 +46,7 @@
                   </PolicyTypeTag>
                 </h3>
 
-                <p>{{ t('policies.collection.description') }}</p>
+                <p>{{ t(`policies.type.${props.currentPolicyType.name}.description`, undefined, { fallback: t('policies.collection.description') }) }}</p>
```

Comment:
The `undefined` here are the interpolated props right? Does it accept `{}`? Kinda looks better if so.

Reply:
It does accept it, but `undefined` is cleaner because it wouldn’t trip a check like `if (values)` and the signature explicitly allows `undefined`.

Reply:
Interested as to why there would be a "trippable" check for `values` somewhere? I take it you mean `Object.values()`? Let me approve this as its not important for the PR.

Reply:
I rather meant `values` as in the name of the argument. Libraries often implement checks on which argument to skip and will `{}` would very likely work, why even let it consider it if it understands `undefined`.

Reply:
> Kinda looks better if so.

Generally, as with `null`, the less I see `undefined` (as opposed to typeof ... "undefined") in an application the better.

Reply:
But why? `typeof thing === 'undefined'` is just more verbose but functionally the same as `thing === undefined`. Same with `null`.

Reply:
I'm not talking about verbosity, I'm not looking for a discussion between `typeof thing === 'undefined'` and `thing === undefined`. I'm talking about setting things to `null` and `undefined`, I just wanted to point out that I wasn't talking about the string `"undefined"` but the value `undefined`